app/models/tipline_message.rb

```diff
@@ -29,6 +31,16 @@ def media_url
     media_url || payload['mediaUrl']
   end
 
+  private
+
+  def verify_user_rate_limit
+    rate_limit = CheckConfig.get('tipline_user_max_messages_per_day', 1500, :integer)
+    # Block tipline user when they have sent more than X messages in 24 hours
+    if self.state == 'received' && TiplineMessage.where(uid: self.uid, created_at: Time.now.ago(1.day)..Time.now, state: 'received').count > rate_limit 
+      Bot::Smooch.block_user(self.uid)
```

Comment:
Will block user generate a log message somewhere so we can find what happened?

Reply:
Yes! Here: https://github.com/meedan/check-api/blob/develop/app/models/concerns/smooch_blocking.rb#L23